vesc_driver/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
 cmake_minimum_required(VERSION 3.5)
 project(vesc_driver)
 
-# Set minimum C++ standard to C++14
-if(NOT "${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}")
-  message(STATUS "Changing CXX_STANDARD from C++98 to C++14")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
-elseif("${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}" STREQUAL "98")
-  message(STATUS "Changing CXX_STANDARD from C++98 to C++14")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-
+# Set minimum C++ standard to C++17
+#if(NOT "${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}")
+#  message(STATUS "Changing CXX_STANDARD from C++98 to C++17")
+#  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
+#elseif("${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}" STREQUAL "98")
+#  message(STATUS "Changing CXX_STANDARD from C++98 to C++17")
+#  set(CMAKE_CXX_STANDARD 17)
+#endif()
+set(CMAKE_CXX_STANDARD 17)
 find_package(ament_cmake_auto REQUIRED)
+find_package(serial_driver REQUIRED)
```

Comment:
This is handled by `ament_auto_find_build_dependencies` below as long as the dependency is in `package.xml` so this line can be removed.

Reply:
This also still has not been fixed.